kroxylicious-systemtests/src/test/java/io/kroxylicious/systemtests/NonJVMClientsST.java

```diff
@@ -223,7 +223,7 @@ void cleanUp() {
     // Skip Kcat execution when architecture is different from x86_64 or amd64 because kcat only provides x86 binaries
     private void checkKcatCompatibility() {
         // Skip Kcat execution when architecture is different from x86_64 or amd64
-        String localArch = System.getProperty("os.arch");
+        String localArch = Environment.ARCHITECTURE;
```

Comment:
When running tests on OCP with different archs, `System.getProperty("os.arch")` returns the arch where system tests are running, not where kroxylicious is deployed